WORKSPACE

```diff
@@ -271,3 +270,90 @@ gcc_register_toolchain(
     name = "gcc_toolchain_x86_64",
     target_arch = ARCHS.x86_64,
 )
+
+# Rust support
+http_archive(
+    name = "rules_rust",
+    integrity = "sha256-ww398ehv1QZQp26mRbOkXy8AZnsGGHpoXpVU4WfKl+4=",
```

Comment:
Is this a newer way of specifying the sha256? Should we change the others too at some point?